src/scripts/main.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const messageBlock = document.createElement('div');
+
+  messageBlock.classList.add('notification', type);
+  messageBlock.style.top = posTop + 'px';
+  messageBlock.style.right = posRight + 'px';
+
+  const theTitle = document.createElement('h2');
+
+  theTitle.classList.add('title');
+  theTitle.innerHTML = title;
+
+  const theDescription = document.createElement('p');
+
+  theDescription.innerText = description;
+
+  messageBlock.append(theTitle, theDescription);
+
+  document.body.append(messageBlock);
+
+  setTimeout(() => {
+    messageBlock.style.visibility = 'hidden';
```

Comment:
```suggestion
    messageBlock.remove();
```

Reply:
It what I had initially, but with remove () it didnt pass the test, since remove() complitelly removed div block from the page and test algoritm can not find it after 4000ms. So I used visibility "hidden".